04-Intro_to_Vue/assignment/html-vue/src/components/contact.vue

```diff
@@ -0,0 +1,25 @@
+<script setup>
+import { ref } from "vue";
+const name = ref ("contact");
+const ingredients = ref(['Email: alex@analyticsltd.com', 'Phone: 720-240-8431', 'Address: 17483 E Temple Dr, Aurora, CO 80015']);
+</script>
+
+<template>
+  <div class="contact">
+    <h1>Contact Page</h1>
+    <main>
+        <ul>
+            <li v-for='ingredient in ingredients' :key='ingredient'>
```

Comment:
Great, this is working. Definitely next step is do something a little more complicated. Like an array of objects like this.

`const ingredients = ref([{protein: 'beef', drink: 'water'}, {protein: 'chicken', drink: 'juice'}])`

and get the component to render each property's value

Something like this.

<img width="206" alt="image" src="https://github.com/user-attachments/assets/1feded89-1ebb-48a8-aacd-76bfbbcc25b3">

Reply:
Something like this:

<script setup>
import { ref } from 'vue'
const contactInfo = ref([
  {
    Email: 'alex_john@analyticsltd.com',
    Phone: '720-240-8431',
    Address: '17483 E Temple Dr, Aurora, CO 80015'
  },
  {
    Email: 'maria_green@analyticsltd.com',
    Phone: '555-555-8431',
    Address: '15555 E Road Dr, Aurora, CO 80015'
  },
  {
    Email: 'sofia_bella@analyticsltd.com',
    Phone: '555-555-8431',
    Address: '22333 E Mitten Dr, Aspen, CO 80015'
  }
])
</script>

<template>
  <div class="contact">
    <h1>Contact Page</h1>
    <h3>Our locations:</h3>
    <main>
      <ul>
        <li v-for="contact in contactInfo" :key="contact">
          <p>Email: {{ contact.Email }}</p>
          <p>Phone: {{ contact.Phone }}</p>
          <p>Address: {{ contact.Address }}</p>
        </li>
      </ul>
    </main>
  </div>
</template>

<style scoped>
.contact {
  text-align: center;
  margin-top: 20px;
  border: 1px #222;
}
</style>

Reply:
![contact vue](https://github.com/user-attachments/assets/b212dc0c-5034-4a5b-809e-685b078f959f)